Windows/DetoursUtils.cpp

```diff
@@ -66,15 +67,15 @@ namespace DetoursUtils
 		if (status == MH_OK)
 		{
 			EngineDevMsg(
-				"Hooked %zu %s function%s.\n",
+				"Hooked %u %s function%s.\n",
 				hook_count,
```

Comment:
Since you removed the `z` qualifier
```suggestion
				static_cast<unsigned>(hook_count),
```